utils/cronjobs_osgeo_lxd/cron_grass8_new_current_src_snapshot.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# script to build GRASS 8.x sources package from the main branch
+# script to build GRASS 8 new_current (8.3) sources package from the main branch
```

Comment:
The "new current" thing is already on the name, is it necessary to repeat it here with different spelling than in the readme file?

GRASS 8 bla bla (8.3) - see RFC: Version Numbering > Usage > "Version should be considered a separate item from the name. The name of the project and software is GRASS GIS, not GRASS GIS 8." Or in other words, major version is repeated twice here.

Reply:
Yes, that's much simpler now!